addons/io_hubs_addon/io/gltf_exporter.py

```diff
@@ -28,6 +28,11 @@ def patched_gather_gltf(exporter, export_settings):
 def get_version_string():
     from .. import (bl_info)
     return str(bl_info['version'][0]) + '.' + str(bl_info['version'][1]) + '.' + str(bl_info['version'][2])
+    #  or (shorter):
```

Comment:
If there is a faster version of this I'm ok with replacing it. If you have a strong opinion about this go ahead and replace and remove the comments.